databricks/koalas/plot.py

```diff
@@ -1118,6 +1118,77 @@ def _plot(data, x=None, y=None, subplots=False, ax=None, kind="line", **kwds):
     return plot_obj.result
 
 
+_backends = {}
+import importlib
+
+
+def _find_backend(backend: str):
+    """
+    Find a pandas plotting backend
+    """
+    # function copied from pandas.plotting._core
+
+    import pkg_resources  # Delay import for performance.
+
+    for entry_point in pkg_resources.iter_entry_points("koalas_plotting_backends"):
+        if entry_point.name == "matplotlib":
+            # matplotlib is an optional dependency. When
+            # missing, this would raise.
+            continue
+        _backends[entry_point.name] = entry_point.load()
+
+    try:
+        return _backends[backend]
```

Comment:
Can we check if "backend" is in "_backends" and raise an `Exception` manually rather than using `try` ??

Reply:
We use the `try` because the `_backends` initially does not have any modules. `_backends` gets modules assigned to it when the modules are loaded, upon the user's request of the plotting module.  The use of:
```python
        return _backends[backend]
``` 

is that if the use has the module already saved in the `_backends` dict, there will be no need for reimporting the package. The image below will example better:
![image](https://user-images.githubusercontent.com/23381512/87014800-44dc0780-c1ea-11ea-8660-f5ab231be0a2.png)

Reply:
@DumbMachine Thanks for the explanation. let's keep this as it as